evalml/tests/automl_tests/test_automl_search_classification.py

```diff
@@ -765,3 +765,31 @@ def test_automl_time_series_classification_pickle_generated_pipeline(mock_binary
     for i, row in a.rankings.iterrows():
         assert a.get_pipeline(row['id']).__class__ == pipeline
         assert pickle.loads(pickle.dumps(a.get_pipeline(row['id'])))
+
+
+@pytest.mark.parametrize("objective", ['F1', 'Log Loss Binary'])
+@pytest.mark.parametrize("optimize", [True, False])
+@patch('evalml.objectives.BinaryClassificationObjective.optimize_threshold')
+@patch('evalml.pipelines.TimeSeriesBinaryClassificationPipeline.predict_proba')
+@patch('evalml.pipelines.TimeSeriesBinaryClassificationPipeline.score')
+@patch('evalml.pipelines.TimeSeriesBinaryClassificationPipeline.fit')
+def test_automl_time_series_classification_threshold(mock_binary_fit, mock_binary_score, mock_predict_proba, mock_optimize_threshold,
```

Comment:
Can you mock `split_data` and assert it's called? If we ever change how we split the data for tuning, I'd like us to know this may not pass anymore.